target_chains/solana/cli/src/main.rs

```diff
@@ -37,90 +59,208 @@ use {
         },
         Account,
         Config as WormholeConfig,
-        GuardianSet,
+        GuardianSet as WormholeSolanaGuardianSet,
         VAA as WormholeSolanaVAA,
     },
 };
 
+#[derive(Default, AnchorSerialize, AnchorDeserialize, Clone, PartialEq, Eq)]
+pub struct GuardianSet {
```

Comment:
can you please document why this struct is necessary? You said it in the github comment, but that should go in the code for posterity.